tests/dualtor_io/test_tor_bgp_failure.py

```diff
@@ -97,16 +97,14 @@ def test_active_tor_kill_bgpd_downstream_standby(
     Action: Shutdown all BGP sessions on the active ToR
     Expectation:
         Verify packet flow after the active ToR (A) loses BGP sessions
-        T1 switch continues to receive IP-in-IP traffic, from lower to upper ToR
-        No switchover occurs
-        verify traffic interruption is < 1 second
+        Verify ToR A standby, ToR B active
+        Verify traffic interruption is < 1 second
     '''
-    with tunnel_traffic_monitor(lower_tor_host, existing=True):
```

Comment:
Why is tunnel traffic verification removed?

Reply:
Newer 202012 images include a mechanism to initiate switchover if BGP goes down. Thus, at the beginning of the test the standby should be forwarding downstream packets to the active via IPinIP. However, once the test kills the bgpd process on the active, a switchover should be triggered which means the old standby is now the new active, and should not be sending any IPinIP traffic, i.e. we would see tunnel traffic for the first part of the test only. I don't believe the tunnel monitor has a way to account for this state change in the middle of the test, which is why I removed it.